ports/libbson/vcpkg.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "libbson",
+  "version": "1.17.6",
+  "description": "libbson is a library providing useful routines related to building, parsing, and iterating BSON documents.",
+  "homepage": "https://github.com/mongodb/libbson",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
```

```suggestion
    {
      "name": "vcpkg-cmake",
      "host": true
    },
    {
      "name": "vcpkg-cmake-config",
      "host": true
    }
```
If we use `vcpkg_cmake_configure()` and `vcpkg_cmake_install()`, `vcpkg-cmake` is also required here.